migration/idempotent/008-tracing-functions.sql

```diff
@@ -636,6 +636,8 @@ BEGIN
                 'IMMUTABLE PARALLEL SAFE STRICT '   ||
                 'AS $f$jsonb_subscript_handler$f$;'
             ;
+
+        ALTER TYPE ps_trace.tag_map SET (SUBSCRIPT = ps_trace.tag_map_subscript_handler);
```

Comment:
check the `typsubscript` column of the `pg_type` table for the `tag_map` type, and only alter the type if the subscript is missing

Reply:
I considered this but didn't come up with a good reason for this. Will do